.github/workflows/ci.yml

```diff
@@ -723,22 +825,25 @@ jobs:
     permissions:
       id-token: write
     steps:
-      - uses: actions/checkout@master
+      - uses: actions/checkout@v4
```

Comment:
Do we need the:
```
        with:
          ref: ${{ github.event.pull_request.head.sha }}
```
or not because on release you are pulling from the tagged commit?

Reply:
This doesn't run on PRs so we can leave it alone.